code/modules/photography/camera/camera_image_capturing.dm

```diff
@@ -11,8 +12,14 @@
 			var/atom/movable/AM = A
 			_step_x = AM.step_x
 			_step_y = AM.step_y
+			is_orbiting = AM.orbiting ? TRUE : FALSE
 	. = ..()
 
+// airlocks don't have actual overlays. make fake overlays temporary
+/image/photo/airlock/New(location, obj/machinery/door/airlock/airlock)
+	. = ..()
+	add_overlay(airlock.get_overlays_for_photo())
```

Comment:
it's because they use vis_contents. you can manually add the vis_contents as an overlay in a cleaner way OR alternatively change airlocks so they don't do this stupidity (the better solution)